src/Microsoft.Extensions.Caching.SqlServer/SqlServerCache.cs

```diff
@@ -164,10 +173,7 @@ public void Set(string key, byte[] value, DistributedCacheEntryOptions options)
                 throw new ArgumentNullException(nameof(value));
             }
 
-            if (options == null)
```

Comment:
Looks like we need to add the null checks to redis cache or it will null ref in GetAbsoluteExpiration